doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -223,7 +206,48 @@ The trained torch model can be extracted for use within the same Python program
 .. code-block::
 
     trainer.train()
-    model = trainer.get_model()
+    model = trainer.get_model()  # will return multiple models if the model_creator does so.
+
+Mixed Precision (FP16) Training
+-------------------------------
+
+You can enable mixed precision training for PyTorch by the ``use_fp16`` flag. This automatically converts the model(s) and optimizer(s) to train using mixed-precision. This requires NVIDIA ``Apex``, which can be installed from `the NVIDIA/Apex repository <https://github.com/NVIDIA/apex#quick-start>`_:
+
+.. code-block::
+    :emphasize-lines: 7
+
+    trainer = PyTorchTrainer(
+        model_creator,
+        data_creator,
+        optimizer_creator,
+        loss_creator=nn.MSELoss,
+        num_replicas=4,
+        use_fp16=True
+    )
+
+``Apex`` is a Pytorch extension with NVIDIA-maintained utilities to streamline mixed precision and distributed training. When ``use_fp16=True``,
+you should not manually cast your model or data to ``.half()``. The flag informs the Trainer to call ``amp.initialize`` on the created models and optimizers and optimize using the scaled loss: ``amp.scale_loss(loss, optimizer)``.
+
+To specify particular parameters for ``amp.initialize``, you can use the ``apex_args`` field for the PyTorchTrainer constructor:
```

Comment:
Should there be a link/explanation of what to pass in for apex_args? Maybe we can add a link to this: https://nvidia.github.io/apex/amp.html#module-apex.amp